specs/Deployment/DeploymentAPI.md

```diff
@@ -167,12 +183,13 @@ they shut down, to refer to the updated framework package.
         // These should be run on a separate thread so as not to hang your app while the
         // packages deploy.
         var initializeTask = Task.Run(() => DeploymentManager.Initialize(deploymentInitializeOptions));
+        //...do other work while the repair is running...
```

Comment:
The comment says "repair", but the method we called was "Initialize".

Either the code should be "Repair", to match the comment; or the comment should say "initialize" to match the code.